ui/app/AppLayouts/Wallet/AddCustomTokenModal.qml

```diff
@@ -34,31 +53,43 @@ ModalPopup {
         label: qsTrId("name")
     }
 
-    Input {
-        id: symbolInput
+
+    Item {
+        width: 200
```

Comment:
Is 200 something special or it should follow the parent width or something else?

Reply:
Actually, on second look, the Item seems useless. You can set the width on the `Input` directly if you want (you might need to set the `right` anchors as `undefined` first)